Overview.src.html

```diff
@@ -3391,21 +3456,23 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
   <dl class=switch>
    <dt><code data-anolis-spec=fileapi>Blob</code>
    <dd>
-    <p>Push a copy of <var>object</var>'s contents to <var>stream</var>.
+    <p><span title=concept-enqueue-ReadableStream>Enqueue</span> a <code>Uint8Array</code> object
```

Comment:
1. Let _buffers_ be a list of ArrayBuffer objects so that the concatenation of their contents equals to _object_'s contents.
2. For each _buffer_ in _buffers_:
 - Enqueue a Uint8Array object wrapping _buffer_.

What do you think about this?
Chrome loads a blob asynchronously: Is "in parallel" needed to allow such implementation?